### DISCUSSION TITLE
Where is the executable?

### DISCUSSION THREAD

**Discussion by RubberCrowy** [AssetRipper/AssetRipper#1576] [Q&A] [answered]
I can't find the executable anywhere. I have no clue how I'm supposed to even start this!

**Comment by ds5678** [comment, marked as answer]
It sounds like you downloaded source code.

**Reply by RubberCrowy** [reply]
Thank you! Evidently, I am not that used to downloading off of Github.

**Comment by ds5678** [comment]
https://github.com/AssetRipper/AssetRipper/releases
